vue-frontend/src/components/partials/ScreenHeaderV2.vue

```diff
@@ -66,37 +67,43 @@ export default {
           url: "/",
           className: "nav-link is-animation-tab",
           target: false,
+          isActive: true,
```

Comment:
Why all are active?

Reply:
Because remove only portfolio link

Reply:
Why do we need to add extra vars for that? Can't we comment portfolio link?

Reply:
I think the flag will be better than the commented code

Reply:
It's fine to comment if we will need it.  Just add to do comment.

Reply:
okay